AppInspector.CLI/Program.cs

```diff
@@ -150,6 +152,16 @@ private static int VerifyOutputArgsRun(CLIPackRulesCmdOptions options)
             return RunPackRulesCommand(options);
         }
 
+        private static int VerifyOutputArgsRun(CLIGetTagsCommandOptions options)
+        {
+            Logger logger = Utils.SetupLogging(options, true);
+            WriteOnce.Log = logger;
```

Comment:
Separate bug, but any error logs are written to the Application Inspector install dir, not the current directory.

Reply:
Is this just for GetTags or in the analyze command too?  Install dir or run dir?

Reply:
I see now. Fixed.